pywwt/annotation.py

```diff
@@ -52,11 +53,42 @@ class Circle(Annotation):
 
     shape = 'circle'
 
-    fill = Bool(False, help='Whether or not the circle should be filled').tag(wwt='fill', sync=True)
-    fill_color = Unicode('white', help='Assigns fill color for the circle').tag(wwt='fillColor', sync=True)
-    line_color = Unicode('white', help='Assigns line color for the circle').tag(wwt='lineColor', sync=True)
-    line_width = AstropyQuantity(1 * u.pixel, help='Assigns line width (in pixels)').tag(wwt='lineWidth', sync=True)
-    radius     = AstropyQuantity(10 * u.pixel, help='Sets the radius for the circle').tag(wwt='radius', sync=True)
+    fill = Bool(False, help='Whether or not the circle should be filled (:class:`bool`)').tag(wwt='fill', sync=True)
+    fill_color = Any('w', help='Assigns fill color for the circle (:class:`str` or `tuple`)').tag(wwt='fillColor', sync=True)
+    line_color = Any('w', help='Assigns line color for the circle (:class:`str` or `tuple`)').tag(wwt='lineColor', sync=True)
+    line_width = AstropyQuantity(1 * u.pixel, help='Assigns line width in pixels (:class:`~astropy.units.Quantity`)').tag(wwt='lineWidth', sync=True)
+    radius     = AstropyQuantity(10 * u.pixel, help='Sets the radius for the circle (:class:`~astropy.units.Quantity`)').tag(wwt='radius', sync=True)
+    sky_relative = Bool(False, help='Whether the size of the circle is relative (in pixels) or absolute (in arcsec) (:class:`~astropy.units.Quantity`)').tag(wwt='skyRelative', sync=True)
+
+    @validate('fill_color')
+    def _validate_fillcolor(self, proposal):
+        if isinstance(proposal['value'], str):
+            return colors.to_hex(proposal['value'])
+        elif isinstance(proposal['value'], tuple):
+            if len(proposal['value']) == 3:
+                return colors.to_hex(proposal['value'])
+            if len(proposal['value']) == 4:
+                self.parent._send_msg(event='annotation_set',
```

Comment:
Instead of doing this, I think you should set ``self.opacity = ...`` otherwise ``circle.opacity`` will be out of sync with the actual opacity in WWT.